### DISCUSSION TITLE
Dose the Redux support the latest version Rect 19 RC?

### DISCUSSION THREAD

**Discussion by BertramYe** [reduxjs/react-redux#2193] [Q&A]
hello when I try install and use the Redux in the latest React 19 rc , but failed, so just have a question, dose the Redux supoort the latest React 19 RC?
![image](https://github.com/user-attachments/assets/625c7199-850a-4ca2-a0de-b7afb98dad86)

**Comment by markerikson** [comment]
Runtime-wise, it'll work fine. We haven't yet published a version that specifically lists React 19 as a peer dependency, so NPM is complaining.  You can override it and it'll be fine.
